atomic_reactor/util.py

```diff
@@ -738,6 +738,41 @@ def __init__(self, registry, insecure=False, dockercfg_path=None, access=None):
 
         self.session = get_retrying_requests_session()
 
+    @classmethod
+    def create_from_config(cls, registry, workflow, access=None):
```

Comment:
IMHO `registry=None` should be defined. We may not have always registry part available, default registry should be used in this case

Reply:
Got it, what about the case when the registry is defined but does not match any pull_registries or source_registry? Currently I just use source_registry in that case, is that correct? Or should it use default configuration `{'uri': registry, 'insecure': False, 'dockercfg_path': None}`

Reply:
Keep current behavior, so default source_registry.

Also you can open ticket about failing when registry doesn't match.